Tests/PerceptionTests/PerceptionTests.swift

```diff
@@ -30,6 +30,34 @@ final class PerceptionTests: XCTestCase {
     self.render(FeatureView())
   }
 
+  func testRuntimeWarning_NotInPerceptionBody_InSwiftUIBody_Wrapper() {
+    self.expectFailure()
+
+    struct FeatureView: View {
+      let model = Model()
+      var body: some View {
+        Wrapper {
+          Text(self.model.count.description)
+        }
```

Comment:
I added a new test to make sure things work when accessing state inside a view not defined in the `SwiftUI` module.